app-ios/tutanota/Sources/Sharing/SharingStorage.swift

```diff
@@ -0,0 +1,176 @@
+import Foundation
+
+let TUTANOTA_APP_GROUP = "group.de.tutao.tutanota"
+let TUTANOTA_SHARE_SCHEME = "tutashare://"
+
+/// this gets shared to the main app and contains all the info
+/// to create a new mail & the cleanup
+struct SharingInfo: Codable {
+  /// this is the key that the information is stored under in the
+  /// UserDefaults and on disk
+  var timestamp: String
```

Comment:
which format is it? Is it better to save a `Date`?

Reply:
it's a kernel clock timestamp, literally an UInt64. No relation to wall time

Reply:
made clear that the value has no meaning